modules/apache-airflow/templates/values.yaml

```diff
@@ -25,6 +25,20 @@ fullnameOverride: ""
 # Provide a name to substitute for the name of the chart
 nameOverride: ""
 
+# Use standard naming for all resources using airflow.fullname template
```

Comment:
These are not new changes (Git is showing them in the diff for some reason). Please ignore changes here.